boards/arm/usb_kw24d512/Makefile.board

```diff
@@ -0,0 +1,7 @@
+FLASH_SCRIPT = openocd.sh
+DEBUG_SCRIPT = openocd.sh
```

Comment:
OpenOCD supports KW21D and KW22D512 devices. As far as I know none uses the KW24D5 :smile:, it has the same functionality as KW22D5, but a different ID. For the KW24D5 I quickly made a patch (https://github.com/jfischer-phytec-iot/openocd/commit/2ec86a443d452135b5c2a1711f99f3dd423397f5). I do not have a jlink, we have built the similar device with KW22D512 and we use the mbed firmware for our debuggers and kits.